### PR TITLE
#258 add example section for member

### PR DESCRIPTION
**Pull request by ansyral**
@vwxyzh @superyyrrzz @chenkennt @vicancy 

not sure whether current position of example section is ok

### CODE REVIEW

File: src/docfx.website.themes/default/partials/class.tmpl.partial [hunk 1]
```diff
@@ -114,6 +114,12 @@
 <h5 id="{{id}}_remarks">{{__global.remarks}}</h5>
 <div class="markdown level1 remarks">{{{remarks}}}</div>
 {{/remarks}}
+{{#example.0}}
+<h5 id="{{id}}_examples">{{__global.examples}}</h5>
```
Comment (hellosnow) [inline src/docfx.website.themes/default/partials/class.tmpl.partial]: close example.0 in case mustache fall back to 0th element?

**Comment by vwxyzh** [comment]
👍

**Comment by hellosnow** [comment]
:+1:

**Comment by superyyrrzz** [comment]
:+1:

**Pull request merged**